### PR TITLE
Minor cosmetic edits

### PR DESCRIPTION
**Pull request by djtango**

### CODE REVIEW

File: twilio.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    @client.account.messages.create({
+      :from => '+441438300248',
+      :to => phone_number,
+      :body => @message,
+    })
+  end
+
+
+  def message_build
+    @message = "Hello customer! Your order for the following has been received: \n#{@receipt}The total of your order came to £#{@total} and it will be with you by #{delivery_promise}. Thank you for eating at Deon Dumplings, your favourite double Ds!"
```
Comment (djtango) [inline twilio.rb]: Objection to this violation - the string would need to be constructed somewhere.

File: takeaway-challenge.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
+require './lib/dish'
```
Comment (djtango) [inline takeaway-challenge.rb]: where hasn't snake case been used?

**Comment by trwh** [comment]
:checkered_flag:

**Comment by djtango** [comment]
recommit

**Pull request closed without merging**